Scripts/Actions/DebugMenuActionsPage.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+namespace DUCK.DebugMenu.Actions
+{
+	public class DebugMenuActionsPage : MonoBehaviour
+	{
+		[SerializeField]
+		private string backCharacter = "←";
+
+		[SerializeField]
+		private string folderArrowCharacter = "▶";
+
+		[SerializeField]
+		private Button actionButtonTemplate;
+
+		[SerializeField]
+		private DebugMenu debugMenu;
+
+		private DebugMenuItemNode rootNode;
+		private DebugMenuItemNode currentPageNode;
+
+		private void Awake()
```

Comment:
This doesn't get the awake call right away since this object is disabled to begin with, we must manually initialize it